judy/src/App.js

```diff
@@ -0,0 +1,106 @@
+import './App.css';
+import React, { Component } from 'react';
+import Navbar from './components/Navbar/Navbar';
+import Intro from './components/Intro'
```

Comment:
great job making all of these components and getting the navbar to conditionally render these different pages!